web/src/components/SubmissionsPairsTable.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+  <v-data-table
+    :headers="headers"
+    :items="items"
+    :footer-props="footerProps"
+    sort-by="similarity"
+    sort-desc
+    must-sort
+    fixed-header
+    dense
+  >
+    <template #item.name="{ item }">
+      <div class="submission-name">
+        <span>{{ item.name }}</span>
+      </div>
+    </template>
+
+    <template #item.label="{ item }">
+      <span class="submission-label">
+        <LabelDot :label="item.label.label" :color="item.label.color" />
+        <LabelText :label="item.label.label" />
+      </span>
+    </template>
+
+    <template #item.timestamp="{ item }">
+      <span class="submission-timestamp">
+        <file-timestamp :timestamp="item.timestamp" />
+      </span>
+    </template>
+
+    <template #item.similarity="{ item }">
+      <span class="submission-similarity">
+        <similarity-display :similarity="item.similarity" progress dense dim-below-cutoff />
+      </span>
+    </template>
+
+    <template #item.cluster="{ item }">
+      <v-tooltip top>
+        <template #activator="{ on, attrs }">
+          <v-btn
+            v-if="item.cluster !== ClusterRelation.NONE"
+            v-bind="attrs"
+            v-on="on"
+            to="/graph"
+            :color="item.cluster === ClusterRelation.SAME ? 'primary' : ''"
+            icon
+            small
+          >
+            <v-icon v-if="item.cluster === ClusterRelation.SAME">mdi-circle-multiple</v-icon>
+            <v-icon v-if="item.cluster === ClusterRelation.DIFFERENT">mdi-circle-multiple-outline</v-icon>
+          </v-btn>
+        </template>
+
+        <span v-if="item.cluster === ClusterRelation.SAME">
+          In same cluster as the current submission.
+        </span>
+
+        <span v-else-if="item.cluster === ClusterRelation.DIFFERENT">
+          In different cluster than the current submission.
+        </span>
+      </v-tooltip>
+    </template>
+
+    <template #item.actions="{ item }">
+      <v-tooltip top>
+        <template #activator="{ on, attrs }">
+          <v-btn icon small v-bind="attrs" v-on="on" :to="`/submissions/${item.fileId}`">
+            <v-icon>mdi-eye-outline</v-icon>
+          </v-btn>
+        </template>
+
+        View submission
+      </v-tooltip>
+
+      <v-tooltip top>
+        <template #activator="{ on, attrs }">
+          <v-btn class="ml-2" icon small v-bind="attrs" v-on="on" :to="`/pairs/${item.id}`">
+            <v-icon>mdi-compare-horizontal</v-icon>

```

Comment:
I'm not convinced by this icon to go to the compare page. Maybe we can re-use the `file-compare` icon we're already using in the compare page itself?

Reply:
I don't think this icon is suitable. It looks too much like a file and can be confusing with the submission.

I do think we should change the eye to same file icon (but with only one file instead of 2) as the submission page.

Reply:
I don't think I agree that it looks too much like a file, it has a clear separation showing two files:
![image](https://user-images.githubusercontent.com/3226995/186391308-ad699883-e895-4587-9010-de0900d43b62.png)

Another possible value is the `set-center` icon, since it is also used on the compare page:
![image](https://user-images.githubusercontent.com/3226995/186389411-1ffbd5b5-c376-4411-a137-59907fedff3d.png)
Or, some kind of search icon, to signal closer inspection:
![image](https://user-images.githubusercontent.com/3226995/186390368-a4b3f16c-563e-4de7-8acd-9a3a5f49d620.png)
But that will also be confusing with showing the file itself.

The main reason why I think the current icon `compare-horizontal` doesn't really fit, is because we don't use it anywhere. It might as well mean swapping the current submission with that one (thus, essentially the same meaning as the other action). So it would maybe also help to re-use an icon on the pairs table so its meaning is consistent throughout the application.


For the eye: we can also make the whole row clickable, like we do with the other tables in the application.

Reply:
From the options listed above `set-center` seems the best to me.

I will also switch to making the rows clickable, although I don't know if having a click action on top of a clickable row is very good UX?

Reply:
Maybe as the row is now clickable we can replace the icon button by an actual button (as there is only one action left now):

![image](https://user-images.githubusercontent.com/6013068/186418646-07d7e5d5-6aab-4134-80fd-aeadd47db47f.png)